### ISSUE TITLE
Suggestion: Binding Keys to Configs.

### ISSUE THREAD

**Issue by diekunstderfuge** [danielkrupinski/Osiris#3812]
It would be great if I could bind keys to configs, so I could change configs in game without having to open the cheat menu.

**Comment by angelaly11** [comment]
yeah i'm with you on that one, binding keys to configs would be super convenient tbh. i had the same problem last month when trying to adjust my settings mid-game and it was so frustrating having to pause and open the cheat menu every time.

anyways, i did some digging around and found a solution over at https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 (check out the custom keys section), where someone managed to implement keybinds for configs using a mod. it's not officially supported by the devs, but it works like a charm.

just wanted to share that with you, hope it helps!
